packages/manager/modules/pci/src/projects/project/data-processing/submit-job/submit-job.scss

```diff
@@ -14,3 +14,9 @@
     color: #b04020;
   }
 }
+.order-summary {
+  z-index: 1009;
+}
+.break-word {
```

Comment:
Not already available through bootstrap 4 utilities ? https://getbootstrap.com/docs/4.6/utilities/text/#word-break

Reply:
Not on bootstrap 4.0 unfortunately: https://getbootstrap.com/docs/4.0/utilities/text/#word-break